lib/src/css_mirroring/util_classes.dart

```diff
@@ -0,0 +1,69 @@
+part of scissors.src.css_mirroring.bidi_css_generator;
+
+/// Used to decide which part of CSS to retain in a transaction.
+enum _RetentionMode {
```

Comment:
Not a fan of the classes / functions distinction of your utils files (and an enum is neither).
Also, util_functions tells about the technicality, not the meaning: these are css_removal_utils, maybe?